Auth0/Authentication.swift

```diff
@@ -223,6 +224,7 @@ public protocol Authentication: Trackable, Loggable {
 
      - returns: an authentication request that will yield Auth0 user credentials after creating the user.
      */
+    @available(*, deprecated, message: "see createUser(email: String, username: String?, password: String, connection: String, userMetadata: [String: Any]?)")
```

Comment:
This is not enough since we should tell them to chain the calls right? Why dont we add more info in the comment?